local_planner/src/nodes/common.h

```diff
@@ -73,7 +73,14 @@ PolarPoint cartesianToPolar(double x, double y, double z,
 **/
 
 Eigen::Vector2i polarToHistogramIndex(const PolarPoint& p_pol, int res);
-
+/**
+* @brief     support function for polarToHistogramIndex
+*            when abs(elevation) > 90, wrap elevation angle into valid
+*            region and azimuth angle changes for +/-180 deg each time
+* @param[in/out]p_pol Polar point with elevation angle [-90,90) and
+*            azimuth angle [-180,180)
+**/
+void wrapPolar(PolarPoint& p_pol);
```

Comment:
There are no tests for wrapPolar directly, I think this would be a pretty easy test to add to help make sure that it works as expected so you don't have to test it in the index conversion tests.

Reply:
Agree there should be a new test for this function. I added a test for wrapPolar, but I left the wrapping index tests.